knative-operator/pkg/webhook/knativeeventing/webhook_validating_test.go

```diff
@@ -1,66 +1,77 @@
-package knativeeventing_test
+package knativeeventing
 
 import (
 	"context"
 	"os"
 	"testing"
 
 	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/apis"
-	. "github.com/openshift-knative/serverless-operator/knative-operator/pkg/webhook/knativeeventing"
+	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/webhook/testutil"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/client-go/kubernetes/scheme"
 	eventingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission/types"
 )
 
+var (
+	ke1 = &eventingv1alpha1.KnativeEventing{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "ke1",
+		},
+	}
+	ke2 = &eventingv1alpha1.KnativeEventing{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "ke2",
+		},
+	}
+
+	decoder types.Decoder
+)
+
 func init() {
 	apis.AddToScheme(scheme.Scheme)
-}
 
-var ke1 = &eventingv1alpha1.KnativeEventing{
-	ObjectMeta: metav1.ObjectMeta{
-		Name: "ke1",
-	},
-}
-var ke2 = &eventingv1alpha1.KnativeEventing{
-	ObjectMeta: metav1.ObjectMeta{
-		Name: "ke2",
-	},
+	dec, err := admission.NewDecoder(scheme.Scheme)
```

Comment:
It seems that the call will never produce an error:
```
// NewDecoder creates a Decoder given the runtime.Scheme
func NewDecoder(scheme *runtime.Scheme) (types.Decoder, error) {
	return decoder{codecs: serializer.NewCodecFactory(scheme)}, nil
}
```

Reply:
Heh, great catch. Pushed a version that ditches the error handling. I moved it outside of tests since we never expect it to fail so I didn't want to pollute tests. Now it's even better.